### ISSUE TITLE
TakeLast example incorrect

### ISSUE THREAD

**Issue by GoogleCodeExporter** [blyzer/morelinq#90]
```
Changed the result for the example
```

Original issue reported on code.google.com by `davide.l...@gmail.com` on 9 Jul 2014 at 7:20

Attachments:
- [TakeLast.cs.patch](https://storage.googleapis.com/google-code-attachments/morelinq/issue-90/comment-0/TakeLast.cs.patch)

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 9 Jul 2014 at 9:49
- Changed title: **TakeLast example incorrect**
- Changed state: **Accepted**
- Added labels: **Component-Docs**, **Type-Defect**
- Removed labels: **Type-Patch**

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 9 Jul 2014 at 9:54
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c303b0e47176.
```

Original comment by `azizatif` on 9 Jul 2014 at 9:57
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
@davide.lettieri Thanks & source package[1] pushed to NuGet.

[1] https://www.nuget.org/packages/MoreLinq.Source.MoreEnumerable.TakeLast/1.0.3
```

Original comment by `azizatif` on 9 Jul 2014 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision da91392f3b87.
```

Original comment by `azizatif` on 12 Aug 2014 at 10:18
